doc/integrator/tilegeneration.rst.tmpl

```diff
@@ -126,3 +126,13 @@ code::
 
   export AWS_ACCESS_KEY_ID=<access_key_id>
   export AWS_SECRET_ACCESS_KEY=<secret_access_key>
+
+Web Interface
+~~~~~~~~~~~~~
+
+It is possible to run tile generation commands through a web interface located at URL
+``<application main URL>/tiles/admin/``. This interface is protected by a password that
+must be specified in environnement variable C2C_SECRET in file `env.default`.
```

Comment:
"in the environnement "
"in the file"

Reply:
Even with a "proper" name right after?

Reply:
The env.default should not be modified, then it should be moved into the env.project file.

Reply:
@sbrunner I have found the mention to ``env.default`` in https://github.com/camptocamp/c2cgeoportal/blob/master/doc/integrator/c2cwsgiutils.rst
Should I update it as well?

Reply:
@sbrunner I have update the filename as you suggested, as well as a few line above and in the c2cwsgiutils doc. See https://github.com/camptocamp/c2cgeoportal/pull/6165/commits/eb972cf854e89d8a935cbd84d25d1465eac03560

@julsbreakdown I have inverted the words order and added some "the". Is it correct?

Thanks!

Reply:
> Should I update it as well?

Yes thanks :-)